apps/web/pages/api/checkProof/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { createClient, kv, VercelKV } from '@vercel/kv';
+
+enum ProofProjectType {
+  NFTBuilder = 'NFTBuilder',
+  Usernames = 'Usernames',
+}
+
+type RequestBody = {
+  address: `0x${string}`;
+  projectType?: ProofProjectType;
+};
+
+function getKVClient(projectType?: ProofProjectType): VercelKV {
+  switch (projectType) {
+    case ProofProjectType.NFTBuilder:
+      return createClient({
```

Comment:
Instead of using a switch and calling createClient repeatedly let’s create a dictionary so U can just

return createClient(PROJECT_CREDENTIALS[ProofProjectType]);